models/Element/Recyclebin/Item.php

```diff
@@ -358,11 +358,20 @@ function (Concrete $object, Data $fieldDefinition, $property, $currentValue) {
         return $copier->copy($data);
     }
 
-    public function getStoreageFile(): string
+    public function getStorageFile(): string
     {
         return sprintf('%s/%s.psf', $this->getType(), $this->getId());
     }
 
+    /**
+     * @deprecated 12.0.0 Typo in name; use Item::getStorageFile instead
```

Comment:
Thank you for your PR!

LGTM, but theoretically we don't need this deprecation because the class is internal and could be rebased for bug fix branch `11.2`
https://github.com/assemblerc01/pimcore/blob/ce888961ac7dbf960562209c2b36358ea5910a37/models/Element/Recyclebin/Item.php#L37


Alternatively, it could stay as an improvement, but it should say something like `deprecated since pimcore 11.3 and will be removed in 12.0` or as `todo`